src/tactic/ring_exp.lean

```diff
@@ -1482,3 +1482,18 @@ add_tactic_doc
   tags        := ["arithmetic", "simplification", "decision procedure"] }
 
 end tactic.interactive
+
+namespace conv.interactive
+open conv interactive
+open tactic tactic.interactive (ring_exp_eq)
+open tactic.ring_exp (normalize)
+
+local postfix `?`:9001 := optional
+
+meta def ring_exp (red : parse (lean.parser.tk "!")?) : conv unit :=
```

Comment:
Could you add a short doc string, even if it just says that it behaves as `ring_exp`, inside a `conv` block?

Reply:
I've done so, and likewise for `ring` while I was there.